backend/ttrs/views.py

```diff
@@ -150,3 +150,19 @@ class MajorDetail(generics.RetrieveAPIView):
     queryset = Major.objects.all()
     serializer_class = MajorSerializer
     permission_classes = (AllowAny,)
+
+
+class RecommendView(generics.ListCreateAPIView):
+    serializer_class = RecommendSerializer
+    permission_classes = (AllowAny,)
+
+    def get_queryset(self):
+        return []
+
+    def create(self, request, *args, **kwargs):
```

Comment:
Why did you override this method?
It seems to be same to the base class's method.